joku/cogs/games/stocks.py

```diff
@@ -235,21 +235,21 @@ def _identify_stock(self, channels: typing.Sequence[discord.abc.GuildChannel], n
         for user in users:        
             stocks = await ctx.bot.database.get_user_stocks(user)
             # Append a column amount to list users
-            user + [sum(userstock.amount for userstock in stocks]
+            user = [user + [sum(userstock.amount for userstock in stocks]]
             # Append a column worth to list users
-            user + [sum(userstock.amount * userstock.stock.price for userstock in stocks)]
+            user = [user + [sum(userstock.amount * userstock.stock.price for userstock in stocks)]]
                 
-        if what == "amount" or "owned": #order by stock amount
+        if what == ("amount" or "owned"): #order by stock amount
```

Comment:
oh cool! That makes sense as well... I'm starting to like python more and more. I will probably be leaving it as what I already changed it to unless i do have to change more code tode(which seems likely)